index.md

```diff
@@ -12,35 +12,101 @@ First we set up & capture the content, then we render it in the scaffolding belo
-### Easy to use
-Cockpit makes Linux discoverable, allowing sysadmins to easily perform tasks such as starting containers, storage administration, network configuration, inspecting logs and so on.
+## Easy to use
+
+### Discoverable
+Cockpit makes GNU/Linux discoverable. See your server in a web browser and perform system tasks with a mouse. It's easy to start containers, administer storage, configure networks, and inspect logs.
+
+[![Storage screenshot]({{ site.baseurl }}/images/site/screenshot-storage.png)]({{ site.baseurl}}/images/site/screenshot-storage.png)
+{:.screenshot.zoom}
+
+### Designed & tested
+Cockpit is designed with your goals in mind. We also routinely test Cockpit with usability studies to make it work the way you'd expect. As a result, Cockpit gets easier to use all the time.
+
+### Team-friendly
+Cockpit is friendly enough for those new to Linux and useful for seasoned admins too.
+
+### Packages included
+[Installing and running Cockpit]({{ site.baseurl }}/running.html) is simple. It's already included in most of the major distributions.
 
 ---
 
-[![Network screenshot]({{ site.baseurl }}/images/site/screenshot-network.png)]({{ site.baseurl }}/images/site/screenshot-network.png){:.screenshot.zoom}
-### No interference
-Jumping between the terminal and the web tool is no problem. A service started via Cockpit can be stopped via the terminal. Likewise, if an error occurs in the terminal, it can be seen in the Cockpit journal interface.
+## Integrated
+
+### Plays well with others
+Jumping between the terminal and the web tool is no problem.
+
+Keep using the command line, Ansible, and your other favorite tools and add Cockpit to the mix with no issues.
+
+A service started via Cockpit can be stopped in a terminal. Likewise, if an error occurs in a terminal, it's also in Cockpit's journal.
+
+[![Network screenshot]({{ site.baseurl }}/images/site/screenshot-network.png)]({{ site.baseurl }}/images/site/screenshot-network.png)
+{:.screenshot.zoom}
+
+### Sign in like normal
+By default, Cockpit uses [your system's normal user logins and privileges]({{ site.baseurl }}/guide/latest/privileges). You don't need to set up any special accounts. Network-wide logins are also supported through [single-sign-on]({{ site.baseurl }}/guide/latest/sso) and other [authentication]({{ site.baseurl }}/guide/latest/authentication) techniques.
+
+### Self-contained
+You don't have to worry about setting up a webserver just to use Cockpit.
+
+### Uses existing APIs
+Cockpit uses APIs that already exist on the system. It doesn't reinvent subsystems or add a layer of its own tooling.
+
+### Efficient
+
+Cockpit only uses memory and CPU when active. When inactive, there is no extra load on your server.
 
 ---
 
-[![Dashboard screenshot]({{ site.baseurl }}/images/site/screenshot-dashboard.png)]({{ site.baseurl }}/images/site/screenshot-dashboard.png){:.screenshot.zoom}
+## Glanceable
```

Comment:
For me as a non-native speaker, it's the first time I hear "glanceable". It's [not in the dictionary](https://dict.leo.org/englisch-deutsch/glanceable), and also sounds rather synthetic. "easy to use" also isn't an adverb, so I don't think it has to be one for consistency. Maybe "keep an overview"?

Reply:
## Glanceable

Glanceable is in the Oxford English Dictionary: https://en.oxforddictionaries.com/definition/glanceable and it's traced back to at least the 1950s, so it's not Sillycon :wink: Vally slang.

(It's also on Dictionary.com, Wiktionary, and others... but those don't have the weight of OED.)

## Easy-to-use as a compound adjective

There's a good writeup of easy-to-use here:
https://english.stackexchange.com/questions/293323/easy-to-use-versus-easy-to-use

Lots of details in general:
https://en.wikipedia.org/wiki/Compound_modifier

Reply:
Ack, thanks!

Reply:
I changed it to easy-to-use in the intro blurb as a result of our conversation, but left it unhyphenated in the header, as it's kind of implied like "[Cockpit is] easy to use[.]" While it's not consistent, it's (probably) correct.

Other headers aren't 1:1 consistent either with the intro blurb, but the meanings are exactly the same, and they're pretty close in text.